lambda-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sailplane/lambda-utils",
-  "version": "6.0.2",
```

Comment:
This patch version decreases instead of increases?

Reply:
Fixed.